components/plan-build/bin/hab-plan-build.sh

```diff
@@ -1082,7 +1082,7 @@ _validate_deps() {
     warn "release in the full dependency chain:"
     warn
     echo "$dupes" | while read dupe; do
-      warn "  * $dupe ( $(echo "$tdeps" | grep "$dupe" | tr '\n' ' '))"
+      warn "  * $dupe ( $(echo "$tdeps" | grep "$dupe/" | tr '\n' ' '))"
```

Comment:
It might be more clear to grep for `"^${dupe}/"` instead. That will explicitly anchor the regex at the beginning of the identifier, and the `${...}` variable delimiter makes clear the distinction between the variable and the rest of the search string.

It took me a second to realize that the `/` character is the slash between the name and version of a fully-qualified identifier, and not a regular expression special character.